### PR TITLE
added dates to Sumerian deprecation

### PR DESCRIPTION
**Pull request by abdallahshaban557**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [X] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [X] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [X] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
      _ref: MDX: `[link](https://link.com)`
            HTML: `<a href="https://link.com">link</a>`_
            
### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/lib/auth/common/overview.mdx [hunk 1]
```diff
@@ -1,34 +1,34 @@
-*Authentication* is a process to validate **who you are** (abbreviated as *AuthN*). The system which does this validation is referred to as an **Identity Provider** or **IdP**. This can be your own self-hosted IdP or a cloud service. Oftentimes, this IdP is a social provider such as Facebook, Google, Amazon, or Sign in with Apple.
```
Comment (sobolk) [inline src/fragments/lib/auth/common/overview.mdx:1]: are those changes intentional ?

File: src/fragments/lib/xr/getting-started.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Callout warning>
 
-The XR category is on a deprecation path. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
+The XR category is on a deprecation path and your Sumerian scenes will not be accessible by 2/21/2023. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples
```
Comment (sobolk) [inline src/fragments/lib/xr/getting-started.mdx:3]:
```suggestion
The XR category is on a deprecation path and your Sumerian scenes will not be accessible after 2/21/2023. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
```

File: src/fragments/lib/xr/sceneapi.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Callout warning>
 
-The XR category is on a deprecation path. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
+The XR category is on a deprecation path and your Sumerian scenes will not be accessible by 2/21/2023. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples
```
Comment (sobolk) [inline src/fragments/lib/xr/sceneapi.mdx:3]:
```suggestion
The XR category is on a deprecation path and your Sumerian scenes will not be accessible after 2/21/2023. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
```

**Review by jimblanc — APPROVED** [review]

File: src/fragments/lib/xr/getting-started.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <Callout warning>
 
-The XR category is on a deprecation path. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
+The XR category is on a deprecation path and your Sumerian scenes will not be accessible by 2/21/2023. You can use BabylonJS, an open source 3D engine to author scenes in the future. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples on using BabylonJS and publishing your scenes with Amplify Hosting.
```
[the remark this answers is not in the document]
Comment (abdallahshaban557) [reply src/fragments/lib/xr/getting-started.mdx:3]: @jhockett - I don't quite understand the feedback here - can you please elaborate on it?

**Review by sobolk — APPROVED** [review]

File: src/fragments/lib/xr/getting-started.mdx [hunk 5]
```diff
@@ -1,7 +1,6 @@
 <Callout warning>
 
-The XR category is on a deprecation path. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
-on using AWS services with Babylon.js
+Your Sumerian scenes will no longer be available as of 2/21/2023. Consequently, the XR category will be unavailable.You can use Babylon.js, an open source 3D engine to author scenes in the future. Please see [Babylon.js sample](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples on using Babylon.js and publishing your scenes with Amplify Hosting.
```
Comment (jhockett) [inline src/fragments/lib/xr/getting-started.mdx:3]:
```suggestion
Your Sumerian scenes will no longer be available as of 2/21/2023. Consequently, the XR category will be unavailable. You can use Babylon.js, an open source 3D engine to author scenes in the future. Please see [Babylon.js sample](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples on using Babylon.js and publishing your scenes with Amplify Hosting.
```

File: src/fragments/lib/xr/sceneapi.mdx [hunk 6]
```diff
@@ -1,7 +1,6 @@
 <Callout warning>
 
-The XR category is on a deprecation path. Please see [https://github.com/aws-samples/aws-tools-for-babylonjs-editor](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples 
-on using AWS services with Babylon.js
+Your Sumerian scenes will no longer be available as of 2/21/2023. Consequently, the XR category will be unavailable.You can use Babylon.js, an open source 3D engine to author scenes in the future. Please see [Babylon.js sample](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples on using Babylon.js and publishing your scenes with Amplify Hosting.
```
Comment (jhockett) [inline src/fragments/lib/xr/sceneapi.mdx:3]:
```suggestion
Your Sumerian scenes will no longer be available as of 2/21/2023. Consequently, the XR category will be unavailable. You can use Babylon.js, an open source 3D engine to author scenes in the future. Please see [Babylon.js sample](https://github.com/aws-samples/aws-tools-for-babylonjs-editor) for examples on using Babylon.js and publishing your scenes with Amplify Hosting.
```

**Review by sobolk — APPROVED** [review]

**Review by jhockett — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**